Fabric.Identity.API/scripts/CatalystDosIdentity.psm1

```diff
@@ -746,22 +745,21 @@ function Edit-ApiRegistration {
         [Parameter(Mandatory=$True)] [string] $accessToken
     )
 
-    $url = "$identityUrl/api/apiresource/$apiName"
+    $url = "$($identityUrl.OriginalString.TrimEnd("/"))/api/apiresource/$apiName"
     $headers = @{"Accept" = "application/json"}
     if($accessToken){
         $headers.Add("Authorization", "Bearer $accessToken")
     }
     try{
-        Invoke-RestMethod -Method Put -uri "$url" -body $body -ContentType "application/json" -Headers $headers | out-null
+        Invoke-RestMethod -Method Put -uri $url -body $body -ContentType "application/json" -Headers $headers | out-null
 
         # Reset api secret
-        $apiResponse = Reset-ApiPassword -identityUrl $url -apiName $apiName -accessToken $accessToken
-        return $apiResponse.apiSecret
+        $apiResponse = Reset-ApiPassword -identityUrl $($identityUrl.OriginalString) -apiName $apiName -accessToken $accessToken
```

Comment:
May need to TrimEnd here as well

Reply:
It is trimming inside of the Reset-ApiPassword